pkgs/applications/science/biology/trimmomatic/default.nix

```diff
@@ -0,0 +1,57 @@
+{ ant
+, fetchFromGitHub
+, jdk11_headless
+, jdk8_headless
+, jre_minimal
+, lib
+, makeWrapper
+, stdenv
+}:
+let
+  my_jre = jre_minimal.override {
```

Comment:
This should be in all-packages.nix

Reply:
OK, I can add it as a callPackage arg if that's better.